docs/ConnectingRealRobot.md

```diff
@@ -1,62 +1,97 @@
-# 1. Supported AS system
+# Connecting real robot
 
-## F Series
+## 1. Supported AS system
+
+### F Series
 
 Handling: Higher than ASF_01000000W
 
-## OpenAS Series
+### OpenAS Series
 
 Handling: Higher than ASE401010XX3S
 
-# 2. How to operate on real robot 
+## 2. Preparation
 
-## 2-1. Preparation
+### 2-1. Robot controller
 
 Make sure that the robot controller used for real-time control satisfies the following conditions.
 
 * The controller is connected with a robot and ready to operate.
-* Nobody is inside the safety fence. 
-* The controller is in [REPEAT] mode. 
+* Nobody is inside the safety fence.
+* The controller is in [REPEAT] mode.
 * TEACH LOCK on the Teach pendant is switched to OFF.
 * Robot is not on Hold status.
 * No error.
 * The controller is connected to an Ubuntu PC with Ethernet cable and both are within the same network subnet.
 
-## 2-2. Start real-time control
+### 2-2. Ubuntu PC
+
+Make sure that the Ubuntu PC used for real-time control satisfies the following conditions.
+
+* The PC is using realtime kernel for Ubuntu 16.04.
+* The user has real-time permissions.
+  * (e.g.)Making a real-time group named `realtime`
+    1. Make a group and add a user
+
+       ```bash
+       sudo addgroup realtime
+       sudo usermod -aG $(whoami)
```

Comment:
I think this line is `sudo usermod -aG realtime $(whoami)`.

Reply:
Thanks, I fixed it.